modules/gcloud/bigquery_test.go

```diff
@@ -78,8 +82,85 @@ func ExampleRunBigQueryContainer() {
 		}
 	}
 
-	fmt.Println(val)
+	// Output:
+	// [30]
+	expectedValue := int64(30)
+	actualValue := val[0]
+	fmt.Println(val[0])
+
+	require.NoError(t, err)
+	if assert.NotNil(t, val) {
+		assert.Equal(t, expectedValue, actualValue)
+	}
+}
+
+func TestBigQueryWithDataYamlFile(t *testing.T) {
+	// runBigQueryContainer {
```

Comment:
This markup is for embedding code snippets into the docs, and that key is probably already in use. Please remove it, alongside its closing mark.